app/services/sms_sender.rb

```diff
@@ -78,15 +77,13 @@ def send_with_send_in_blue
     rescue SibApiV3Sdk::ApiError => e
       # 401 Unauthorized
       # 400 Invalid telephone number
-      add_sib_error_to_breadcrumbs(e)
       handle_failure(error_message: "SendInBlue error: #{e.message} (#{e.code})")
     end
   end
 
   # These errors should not trigger a retry, because it would only fail again
   NETSIZE_PERMANENT_ERRORS = [
     15, # Message concatenation limit exceeded
-    16, # Unable to route message.
```

Comment:
Je commence à avoir des doutes sur le caractère permanent de l'erreur "Unable to route message". J'ai regardé des docs d'autres providers de SMS, et ils ont l'air de dire que le routage de SMS est un processus complexe, et donc je suppose qu'un échec de routage peut être intermittent.

J'ai donc ici retiré cette erreur de notre liste d'erreurs considérées "permanentes".